pyethapp/tests/test_console_service.py

```diff
@@ -163,14 +163,14 @@ def test_console_name_reg_contract(test_app):
         tx_to = b''
         evm_code = solidity.compile(solidity_code)
         chain = test_app.services.chain.chain
-        assert chain.head_candidate.get_balance(tx_to) == 0
+        assert chain.state.get_balance(tx_to) == 0
```

Comment:
Cool, then I think it's better to merge both PR to avoid further conflicts, and fix remaining failures on that base.